src/state-client.ts

```diff
@@ -552,6 +545,11 @@ class StateClient {
   /** @ignore */
   public getExtensionTriviaLeaderboard = (identifer: string) =>
     this.signedRequest<undefined, TriviaLeaderboard>(identifer, 'GET', 'curated_poll_leaderboard');
+
+  /** @ignore */
+  public sendTransactionToServer(identifier: string, tx: Transaction) {
+    this.signedRequest<Transaction, Record<string, unknown>>(identifier, "POST", "bits/transactions", tx);
+  }
```

Comment:
With the braces, this won't automatically return the result of `signedRequest`. You can either remove them and add a `=>` as above, or add an explicit `return` statement before the call to `this.signedRequest`.